pages/common/git-config.md

```diff
@@ -1,27 +1,28 @@
 # git config
 
-> Get and set repository or global options.
+> Manage custom configuration options for git repositories.
+> These configurations can be local (for the current repository) or global (for the current user).
 
-- Print list of options for current repository:
+- List only local configurations (stored in `.git/config` in the current repository):
 
 `git config --list --local`
 
-- Print global list of options, set in ~/.gitconfig:
+- List only global configurations (stored in `~/.gitconfig`):
 
 `git config --list --global`
 
-- Get full list of options:
+- List all configurations that have been defined either locally or globally:
```

Comment:
I don't like calling them "options" because many of them are user-defined, and in particular aliases have both user-specified keys and user-specified values. "Options" feels more like picking from a list, which is definitely is the case for some of the entries, but not all of them.

"Directives" could work, but it's a bit uncommon/fancy -- how about "entries"? This would also be consistent with the other descriptions.